Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DTO.RestRequests;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class EditUserCommand : IEditUserCommand
+    {
+        private readonly IValidator<EditUserRequest> userInfoValidator;
+        private readonly IValidator<EditUserRequest> userPasswordValidator;
+        
+        public EditUserCommand
+            ([FromServices] IValidator<EditUserRequest> userInfoValidator, 
+            [FromServices] IValidator<EditUserRequest> userPasswordValidator)
```

Comment:
What? 2 same validators? 0_0

Reply:
No, they are not the same, but we have to separate validator for edit user request if sometimes we want to send request without password

Reply:
> No, they are not the same, but we have to separate validator for edit user request if sometimes we want to send request without password

Then why they both are` IValidator<EditUserRequest>`, mb there should be `IValidator<EditUserRequest>` ??? How do you want service to know which of the injections to use if you just use the same interface?

Reply:
Wait a minute
Why do you even use 2 validators instead of one? 0_0
You have 1 model which is EditUserRequest

Reply:
fixed